tsinfer/inference.py

```diff
@@ -1128,6 +1138,31 @@ def __init__(
         self.mismatch[:] = mismatch
         self.precision = precision
 
+        if len(recombination) == 0:
+            logger.debug("Fewer than two inference sites: no recombination possible")
```

Comment:
Probably better to put these at ``info`` log level rather than debug. Full debug output is massive (logs info on every matching done)